pkg/vpcmodel/output.go

```diff
@@ -66,7 +71,9 @@ func NewOutputGenerator(c1, c2 MultipleVPCConfigs, grouping bool, uc OutputUseCa
 		nodesConn:      map[string]*VPCConnectivity{},
 		subnetsConn:    map[string]*VPCsubnetConnectivity{},
 	}
-	if !archOnly {
+	graphicFormat := slices.Contains([]OutFormat{DRAWIO, ARCHDRAWIO, SVG, ARCHSVG, HTML, ARCHHTML}, f)
+	archOnlyFormat := slices.Contains([]OutFormat{ARCHDRAWIO, ARCHSVG, ARCHHTML}, f)
```

Comment:
if we will have a non graphical architectural format that, it will be part of the list

Reply:
But this format is graphic, isn't it? Am I missing something?

Reply:
this code is general for all output formats.
in case of not arch - we do connectivity analysis.
for now, all are arch formats are graphic, but it isnt necessary always be the case.